Formula/ipfs.rb

```diff
@@ -1,14 +1,14 @@
 class Ipfs < Formula
```

Comment:
Might be good to just rename this formula?

Reply:
CC @chenrui333

Reply:
Wait, is the command still called `ipfs`? How confusing.

Reply:
We didn't wanted to break *too many things*.

AFAIT At first we wanted to rename and add `alias ipfs=kubo` to people's `.bashrc` but modifying peoples dot file like that is not a nice thing to do. (also there is far too much setups for us to get right everywhere)

So now it's still called `ipfs` and if want you can add `alias kubo=ipfs`.

I think an alias or symlink from `kubo` to `ipfs` would be nice.

Reply:
make sense, @Jorropo thanks for the input!